aspnetcore/fundamentals/minimal-apis/responses.md

```diff
@@ -120,14 +120,28 @@ In order to document this endpoint correctly the extension method `Produces` is
 
 :::code language="csharp" source="~/fundamentals/minimal-apis/9.0-samples/Snippets/Program.cs" id="snippet_04":::
 
-<a name="binr7"></a>
+<a name="binr10"></a>
 
 ### Built-in results
 
 [!INCLUDE [results-helpers](~/fundamentals/minimal-apis/includes/results-helpers.md)]
 
 The following sections demonstrate the usage of the common result helpers.
 
+#### Server-Sent Events (SSE)
```

Comment:
I think this section should come after the JSON built-in result type. And maybe even further down -- it might fit better either before or after the Stream built-in result type.